addons/account/models/partner.py

```diff
@@ -514,6 +514,17 @@ def create(self, vals_list):
                     vals['supplier_rank'] = 1
         return super().create(vals_list)
 
+    def unlink(self):
+        """
+        Prevent the deletion of a partner "Individual", child of a company if:
+        - partner in 'account.move'
+        - state: all states (draft and posted)
+        """
+        moves = self.sudo().with_context(active_test=False).env['account.move'].search_count([('partner_id', '=', self.id), ('state', 'in', ['draft', 'posted'])])
+        if moves:
+            raise UserError(_("Record cannot be deleted. Partner used in 'account.move'"))
```

Comment:
Too technical for users

Reply:
Yep, I just wanted to avoid any confusion if only Invoicing is installed or Purchase and Invoicing are installed or only Accounting, etc.

What about:
"Record cannot be deleted. Partner used in an accounting move" ?

Reply:
> used in Accounting

A "move" is a technical term referring to a Journal Entry/Invoice/Refund, and doesn't mean anything to a customer.